tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -603,26 +604,34 @@ node_modules/chrome-devtools-frontend/front_end/audits2/lighthouse/renderer/deta
 node_modules/chrome-devtools-frontend/front_end/audits2/lighthouse/renderer/details-renderer.js(285,31): error TS2694: Namespace 'DetailsRenderer' has no exported member 'DetailsJSON'.
 node_modules/chrome-devtools-frontend/front_end/audits2/lighthouse/renderer/details-renderer.js(298,8): error TS2339: Property 'DetailsRenderer' does not exist on type 'Window'.
 node_modules/chrome-devtools-frontend/front_end/audits2/lighthouse/renderer/details-renderer.js(303,2): error TS1131: Property or signature expected.
+node_modules/chrome-devtools-frontend/front_end/audits2/lighthouse/renderer/details-renderer.js(303,8): error TS2300: Duplicate identifier 'type'.
```

Comment:
It looks like we badly parsed properties as typedefs when there is a line-beginning asterisk in the middle of the type. [1] Now there is a duplicate error marked. Would be fixed completely by #26528

[1]

```js
/** @typedef {{
 * ohno,
 * an,
 * asterisk
 * }}
 */
```